support/config/featuregates.go

```diff
@@ -13,9 +13,9 @@ func FeatureGates(fg *configv1.FeatureGateSelection) []string {
 		enabled = fg.CustomNoUpgrade.Enabled
 		disabled = fg.CustomNoUpgrade.Disabled
 	} else {
-		fs := configv1.FeatureSets[fg.FeatureSet]
-		for _, fgDescription := range fs.Enabled {
```

Comment:
This is wrong. The function still needs to iterate over enabled featuregates. As it is, it's only iterating over disabled ones.

Reply:
good catch.
thanks, fixed.